kinesisfirehose-logs-extension-demo/README.md

```diff
@@ -0,0 +1,226 @@
+# Centralize log collection with Amazon Kinesis Data Firehose using Lambda Extensions
+
+## Introduction
+
+This pattern walks through an approach to centralize log collection for Lambda function with Kinesis firehose using external extensions. The provided code sample shows how to get send logs directly to Kinesis firehose without sending them to AWS CloudWatch service.
```

Comment:
Suggestion: This example show how to centralize log collection for a Lambda function using Kinesis Data Firehose. The provided code sample uses Lambda extensions to receive logs from Lambda and send them directly to Kinesis Data Firehose without sending them to Amazon CloudWatch service.